main.go

```diff
@@ -31,12 +31,21 @@ type backup struct {
 
 func main() {
 
-	dailyBackupsRetainedPtr := flag.Int("daily-backups", 7, "the amount of daily backups which should be retained")
-	weeklyBackupsRetainedPtr := flag.Int("weekly-backups", 4, "the amount of weekly backups which should be retained")
-	kanisterNamespacePtr := flag.String("kanister-namespace", "kanister", "the kubernetes namespace in which kanister resides")
-	blueprintNamePtr := flag.String("blueprint-name", "postgres-bp", "the blueprint of the backups which should be managed")
-	s3ProfileNamePtr := flag.String("s3-profile", "s3-profile", "the S3 profile of the S3 bucket which contains the backups")
-	evalSchedulePtr := flag.String("eval-schedule", "1/5 * * * *", "The cron schedule at which backups should be evaluated")
+	//get configuration options from environment variables
+	dailyBackupsRetained, err := strconv.Atoi(os.Getenv("DAILY_BACKUPS"))
+	if err != nil {
+		log.Fatalf("Error retrieving DAILY_BACKUPS env var: %v", err)
+	}
+	weeklyBackupsRetained, err := strconv.Atoi(os.Getenv("WEEKLY_BACKUPS"))
+	if err != nil {
+		log.Fatalf("Error retrieving WEEKLY_BACKUPS env var: %v", err)
+	}
+	kanisterNamespace := os.Getenv("KANISTER_NAMESPACE")
```

Comment:
would it be safer to also handle the case where the following three ENV are empty?